src/app/src/pages/Profile.tsx

```diff
@@ -152,6 +160,24 @@ export default function Profile({ currentUser }: Props): JSX.Element {
     fetchData();
   }, [author?.id]);
 
+  // Update isFollowing if the author is being followed by the current user
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const res = api.authors
+          .withId("" + currentUser?.id)
+          .followers.withId("" + author?.id)
+          .isAFollower();
```

Comment:
![Screen Shot 2022-04-03 at 11 37 20 PM](https://user-images.githubusercontent.com/55111012/161480692-0a3b8723-998e-427a-9223-fd3df9db0e08.png)
I think this is backwards.
They follow me and I don't follow them. But they get the follow button and I get unfollow.
I think it needs the same order as your unfollow api call!

Reply:
should work now